library/classes/Installer.class.php

```diff
@@ -146,7 +146,16 @@ public function drop_database() {
   }
 
   public function grant_privileges() {
+	$version =  mysqli_get_server_version($this->dbh);
+	if($version >= 80000){
+		$this->execute_sql("drop user $this->login@$this->loginhost");
+		$this->execute_sql("FLUSH PRIVILEDGES");
```

Comment:
I need to spin up a MySQL-8 to test this. From first glance it might do the trick. Will test this in a while

Reply:
Do it in Docker -- change to `mysql:5.7` to `mysql:8.0` -- just don't commit it

Reply:
Nevermind, code needs to be reworked.